tinkerbell/stack/values.yaml

```
@@ -21,6 +23,24 @@ stack:
     # downloadURL only works with the > 0.8.1 Hook release because
     # previous Hook versions didn't provide a checksum file.
     downloadURL: https://github.com/tinkerbell/hook/releases/download/latest
+    persistence:
+      enabled: true
+      ## type hostpath or pvc
+      # hostpath : only works on a single worker node cluster
+      type: hostpath
+      ## When type is pvc
+      # storageClassName: default
+
+      ## PVC accessModes : 
+      # Require ReadWriteMany in order to download Hook and scale up kubernetes tink-stack Deployment resource.
+      # If ReadWriteOnce is used, you will need to download manually Hook into stack Pod (/usr/share/nginx/html/).
+      accessModes:
+        - ReadWriteMany
+      size: 1Gi
+      # annotations: {}
+      # selectorLabels: {}
```

```suggestion
			# Annotations to include on the PVC object.
      annotations: {}
      
      # An optional selector to narrow the Volumes considered for binding to the PVC.
      # Should be structured the same as a raw PVC selector.
      # See https://kubernetes.io/docs/concepts/storage/persistent-volumes/#selector.
      selector: {}
```

I'm suggesting changing the `selector` so users can define the selector using expressions as well as labels.